compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionOfTensorOps.cpp

```diff
@@ -489,8 +489,8 @@ struct FusionOfTensorOpsPass
             if (!reshapeOp)
               return true;
 
-            return isa<linalg::LinalgOp, tensor::UnPackOp,
-                       LinalgExt::UnsetEncodingOp>(
+            return llvm::isa_and_nonnull<linalg::LinalgOp, tensor::UnPackOp,
+                                         LinalgExt::UnsetEncodingOp>(
                 reshapeOp.getSrc().getDefiningOp());
```

Comment:
I think it is okay to land it like this. We are probably hitting the case that `reshapeOp.getSrc()` is a bbarg, so getDefiningOp returns nullptr.